### PR TITLE
DOCS-815: Expand ML intro copy

### PR DESCRIPTION
**Pull request by andf-viam**
Expand ML landing page intro copy with explanatory description and simple example.

- Maybe just a little too much for the Manage section? Remove the cat pictures example?

### CODE REVIEW

**Review by npentrel — CHANGES_REQUESTED** [review]
good start but I think we want to change this to be more focused on robotics

File: docs/manage/ml/_index.md [hunk 1]
```diff
@@ -5,11 +5,25 @@ weight: 40
...
+
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+To make use of ML models with your robot, you can use the built-in ML model service to train image classification models for object detection and classification or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
+
+### Object detection and classification
+
+Once you have [created the ML model service](/services/ml/#create-an-ml-model-service) for your robot, you can [train image classification models](train-model/) to enable it to detect people, animals, plants or other objects with bounding boxes and perform actions when they are detected.
+
+![Gif of a dog being labeled](/tutorials/img/pet-treat-dispenser/app-data-images.png)
```
Comment (andf-viam) [inline docs/manage/ml/_index.md:23]: Can also use the image Arielle has added as part of https://github.com/viamrobotics/docs/pull/1382/files.

Technically: both pictures are of the Data service showing a bunch of images that will shortly be used to train, not the actual training-of.

File: docs/manage/ml/_index.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ description: "Use Viam's built-in machine learning capabilities to train image c
 # SME: Aaron Casas
 ---
 
+Machine learning is an approach to programming where you provide an algorithm with a large data set, and train it to discover patterns in that data on its own.
```
[the remark this answers is not in the document]
Comment (andf-viam) [reply docs/manage/ml/_index.md]: Thanks! I've based my rewrite strongly off your suggestion. Let me know what you think!

File: docs/manage/ml/_index.md [hunk 3]
```diff
@@ -5,18 +5,25 @@ weight: 40
...
+Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
 
-For example, say you have a collection of pictures of your cat.
-As you train your algorithm on just a few pictures, it might not be able to determine that a picture of your cat's face and a picture of your cat sleeping are the same cat.
-As you train the algorithm on more and more pictures, it might be able to recognize your cat from any angle, or to differentiate between multiple different cats.
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
```
Comment (npentrel) [inline docs/manage/ml/_index.md:15]: I think those would still be tensorflow lite models - just user provided ones, so I think more correct to say:

```suggestion
Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
```

File: docs/manage/ml/_index.md [hunk 4]
```diff
@@ -5,18 +5,25 @@ weight: 40
...
+
+To make use of ML models with your robot, you can use the built-in ML model service to train image classification models for object detection and classification or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
```
Comment (npentrel) [inline docs/manage/ml/_index.md:17]:
```suggestion
To make use of ML models with your robot, you can use the built-in [ML model service](/services/ml/) or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
```

File: docs/manage/ml/_index.md [hunk 5]
```diff
@@ -28,4 +35,5 @@ To view or export captured data, see [Data Management](../data/).
   {{% card link="/manage/ml/train-model" size="small" %}}
   {{% card link="/manage/ml/upload-model" size="small" %}}
   {{% card link="/services/ml" size="small" custom="Deploy Model" %}}
+  {{% card link="/tutorials/projects/pet-treat-dispenser/" size="small" custom="Tutorial: Smart Pet Feeder" %}}
```
Comment (npentrel) [inline docs/manage/ml/_index.md:38]: good addition

File: docs/manage/ml/_index.md [hunk 6]
```diff
@@ -5,18 +5,25 @@ weight: 40
...
-As you train your algorithm on just a few pictures, it might not be able to determine that a picture of your cat's face and a picture of your cat sleeping are the same cat.
-As you train the algorithm on more and more pictures, it might be able to recognize your cat from any angle, or to differentiate between multiple different cats.
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models, but you can use other models as well as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+To make use of ML models with your robot, you can use the built-in ML model service to train image classification models for object detection and classification or create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
+
+### Object detection and classification
+
+Once you have [created the ML model service](/services/ml/#create-an-ml-model-service) for your robot, you can [train image classification models](train-model/) to enable it to detect people, animals, plants or other objects with bounding boxes and perform actions when they are detected.
```
Comment (npentrel) [inline docs/manage/ml/_index.md:21]: you don't need to first create the service to train. The service only deploys.

```suggestion
Once you have [deployed the ML model service](/services/ml/#create-an-ml-model-service) for your robot, you need to add an additional service that makes use of the model.
For object detection and classification you can use the [Vision Service](/services/vision) [mlmodel detector](https://docs.viam.com/services/vision/detection/#configure-a-mlmodel-detector) or [mlmodel classifier](https://docs.viam.com/services/vision/classification/#configure-a-mlmodel-classifier).
For other usage, you can create a [modular resource](/program/extend/modular-resources/) to integrate it with your robot.
```

This is where the modular resource comes in I think

File: docs/manage/ml/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/manage/ml/_index.md:23]: sounds good. can use that until we can create something better

File: docs/manage/ml/_index.md [hunk 7]
```diff
@@ -5,18 +5,25 @@ weight: 40
...
+
+![Gif of a dog being labeled](/tutorials/img/pet-treat-dispenser/app-data-images.png)
+
+When training machine learning models, it is important to supply a variety of different data about the subject. In the case of object detection, it is important to provide images of the object in different situations, such as from different angles or in different lighting situations. The more varied the provided data set, the more accurate the resulting model becomes.
```
Comment (npentrel) [inline docs/manage/ml/_index.md:25]: This should probably now actually move to the train page - it no longer feels like it fits here

File: docs/manage/ml/_index.md [hunk 8]
```diff
@@ -5,18 +5,25 @@ weight: 40
...
 
-You can use Viam's built-in machine learning capabilities to [train image classification models](train-model/) and [deploy these models to your robots](../../services/ml/).
 You can also [upload and use existing models](upload-model/).
```
Comment (npentrel) [inline docs/manage/ml/_index.md:27]: I think this sentence should now go. And the info about the Data Management Service should probably also go or be integrated into the bit where we talk about training inside Viam

File: docs/manage/ml/_index.md (hunk 1 above)
Comment (npentrel) [reply docs/manage/ml/_index.md:23]: could consider https://www.youtube.com/watch?v=CP14LR0Pq64

File: docs/manage/ml/_index.md (hunk 7 above)
Comment (andf-viam) [reply docs/manage/ml/_index.md]: Moved!

File: docs/manage/ml/_index.md (hunk 8 above)
Comment (andf-viam) [reply docs/manage/ml/_index.md]: Agreed, removed both.

**Comment by andf-viam** [comment]
Thanks @npentrel , I've folded in all of your guidance. LMK if you'd like to see any further changes, happy to adjust.

File: docs/manage/ml/_index.md [hunk 9]
```diff
@@ -5,20 +5,36 @@ weight: 40
...
-To capture and synchronize data to the platform, see [Data Management Service](../../services/data/).
-To view or export captured data, see [Data Management](../data/).
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+You can [train your own image classification models](/manage/ml/train-model/) or [add an existing model](/manage/ml/upload-model/) for object detection and classification within the platform using data from the [Data Management Service](../../services/data/).
+Object detection and classification models are commonly used to enable robots to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+
+To make use of ML models with your robot, you can use the built-in [ML model service](/services/ml/) to deploy and run the model.
+
+### Object detection and classification
```
Comment (npentrel) [inline docs/manage/ml/_index.md:22]: The second bullet point isn't about object detection and classification so I think best to remove

```suggestion
```

File: docs/manage/ml/_index.md [hunk 10]
```diff
@@ -5,20 +5,34 @@ weight: 40
...
+
+You can [train your own image classification models](/manage/ml/train-model/) or [add an existing model](/manage/ml/upload-model/) for object detection and classification within the platform using data from the [Data Management Service](../../services/data/).
+Object detection and classification models are commonly used to enable robots to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+
+To make use of ML models with your robot, you can use the built-in [ML model service](/services/ml/) to deploy and run the model.
+
+Once you have [deployed the ML model service](/services/ml/#create-an-ml-model-service) to your robot, you can then add another service to make use of the model.
+
+* For object detection and classification you can use the [Vision Service](/services/vision) [mlmodel detector](https://docs.viam.com/services/vision/detection/#configure-a-mlmodel-detector) or [mlmodel classifier](https://docs.viam.com/services/vision/classification/#configure-a-mlmodel-classifier).
```
Comment (nataliajacobowitz) [inline docs/manage/ml/_index.md:24]: do you want commas for this list of 3 services? line 24

Comment (andf-viam) [reply docs/manage/ml/_index.md:24]: Ah, this is two items but looks like three. Will clarify this, thanks!

**Pull request merged**